docs/knative-offerings.md

```diff
@@ -19,6 +19,7 @@ Here is a list of commercial Knative products (alphabetically):
 - [IBM Cloud Code Engine](https://cloud.ibm.com/codeengine): A fully-managed serverless platform that runs all your containerized workloads, including http-driven application, batch jobs or event-driven functions.
 - [Red Hat Openshift Serverless](https://docs.openshift.com/container-platform/4.6/serverless/serverless-getting-started.html): enables stateful, stateless, and serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
```

Comment:
```suggestion
- [Red Hat Openshift Serverless](https://docs.openshift.com/container-platform/4.7/serverless/serverless-getting-started.html): enables stateful, stateless, and serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
```

Reply:
Would you mind doing me a solid please and adding this suggestion to update the version for OCP? 🙏🏻